src/daemon/rpmostreed-transaction-livefs.c

```diff
@@ -522,6 +522,106 @@ replace_subpath (OstreeRepo *repo,
   return TRUE;
 }
 
+/* The sledgehammer 🔨 approach. Because /usr is a mount point, we can't replace
+ * all of it. We could do a diff, but doing that precisely and quickly depends
+ * on https://github.com/ostreedev/ostree/issues/1224
```

Comment:
Hmm, it's true, we are talking about a lot of `fdopendir`/`readdir`/`stat` calls, though I do still wonder how bad it would be.

Actually, that reminds me of https://github.com/cgwalters/rpm-ostree/pull/2, in which I measured adding a `stat` penalty on a real compose to be around ~60 ms. Of course, it'd be much larger, but even so, even if it's say, 1 second, it seems like an OK tradeoff considering the added safety? E.g. I would be *very* hesitant to ever use this feature on my Atomic Workstation; feels a bit too violent. ⚡️ 🔥 💻 😱